data/json/itemgroups/misc.json

```diff
@@ -148,5 +148,12 @@
     "type": "item_group",
     "//": "Horse vehicle items",
     "items": [ [ "riding_saddle", 50 ], [ "yoke_harness", 50 ] ]
+  },
+  {
+    "id": "washing_detergents",
```

Comment:
Why give all detergent a tag that identifies them as such? There are functions to in the code to query for items with specific tags. That is much easier than keeping a list of items up to date.

Reply:
Alternatively use a requirement. See bionics.cpp `player:has_enough_anesth` - there the game checks the requirement "anesthetic". This logic is also used in other code in iuse.cpp and iexamine.cpp

Reply:
Thanks for the review! I've decided to use a flag.

Reply:
Thanks for the review! I've decided to use a flag.